ogre2/src/Ogre2Scene.cc

```diff
@@ -26,6 +26,7 @@
 #include "ignition/rendering/ogre2/Ogre2Grid.hh"
```

Comment:
You are adding the implementation for Ogre2 too, can you edit the PR message ?

Reply:
done :+1: